src/index.html

```diff
@@ -1,17 +1,169 @@
+        <div class="form-filed">
+          <label for="surname">Surname:
+            <input type="text" 
+            name="surname" 
+            id="surname"
+            autocomplete="off" 
+            required
+            >
+          </label> 
+        </div>
+        <div class="form-filed">
+          <label for="name"> Name:</label>
+          <input type="text" 
+          name="name" 
+          id="name"
+          autocomplete="off"
+          >
+        </div>
+        <div class="form-filed">
+          <label for="age">How old are You?</label>
+          <input type="number"
+            name="age"
+            id="age"
+            pattern="\d+"
+            value="12"
+            min="1"
+            max="100"
+          >
+        </div>
+        <div class="form-filed">
+          <label for="date-birth">Full date of birth:</label>
+          <input type="date"
+            name="date-birth"
+            id="date-birth"
+            placeholder="date"
+          >
+        </div>
+        <div class="form-filed">
+          <label for="accept">I accept the term of the agreement</label>
+          <input type="checkbox"
+            name="accept-agreement"
+            id="accept"
+            value="yes"
+          >
+        </div>
+      </fieldset>
+
+      <fieldset class="form-box">
+        <legend>Registration</legend>
+        <div class="form-filed">
+          <label for="email">E-mail: </label>
+          <input type="email"
+            placeholder="email@example.com"
+            name="email"
+            id="email"
+            autocomplete="off"
+          >
+        </div>
+        <div class="form-filed">
+          <label for="password">Password: </label>
+          <input type="password"
+            name="password"
+            id="password"
+            minlength="5"
+            maxlength="15"
+            autocomplete="off"
+          >
+        </div>
+      </fieldset>
+
+      <fieldset class="form-box">
+        <legend>An interesting fact about you!</legend>
+
+        <div class="form-filed">
+          <label for="cats"
+            >Do you love cats?
+            <input type="radio" 
+            name="cats"  
+            value="Yes"
+            > Yes
+            <input type="radio"
+             name="cats" 
+             value="No"
+            > No
+          </label>
+        </div>
+        <div class="form-filed">
+          <label for="color">What is your favorite color?</label>
+          <input type="color" 
+          name="color" 
+          id="color"
+          >
+        </div>
+        <div class="form-filed">
+          <label for="bed-time">What time do you go to bed?</label>
+          <input type="time" 
+            name="bed-time" 
+            id="bed-time"
+          >
+        </div>
+        <div class="form-filed">
+          <label for="car-select">What are your favorite brand of cars?</label>
+          <select name="car-select" 
```

Comment:
Fix formating and name attribute.
Make sure the label is clicable and focuses the select